client/consensus/common/src/lib.rs

```diff
@@ -13,7 +13,8 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
-//! Collection of consensus specific imlementations
+
+//! Collection of common consensus specific imlementations
```

Comment:
And this?

Reply:
usually just copy that first line into the field for crates (until we have some decent copyedits), and fixed it while I was at it.